scripts/xtp_modify_jobfile.in

```diff
@@ -0,0 +1,191 @@
+                      required=True,
+                      type=argparse.FileType('r'),
+                      default='',
+                      help='Jobfile file to select from.')
+
+progargs.add_argument('-o', '--output',
+                      dest='j_output',
+                      action='store',
+                      required=True,
+                      type=argparse.FileType('w'),
+                      default='',
+                      help='Filename to write new jobfile to.')
+
+progargs.add_argument('-l', '--job_ids',
+                      dest='job_ids',
+                      action='store',
+                      type=str,
+                      nargs="+",
+                      help='Either a list of ids \'1 3 5\' or a range \'1-5\' or a combination thereof')
+
+progargs.add_argument('-s', '--selector',
+                      dest='selector',
+                      action='store',
+                      type=str,
+                      help='A more flexible way to select jobs can be combined with ranges. Specify the xml tag you want inside the job via e.g. \'input/regions/region.id:0\'. The tag must exist and the value behind the colon supports \'?\' and \'*\' wildcards ')
+
+
+OPTIONS = progargs.parse_args()
+
+
+# =============================================================================
+# LXML
+# =============================================================================
+def RepresentsInt(s):
+    try:
+        int(s)
+        return True
+    except ValueError:
+        return False
+
+
+def parseIds(parseroutput):
+
+    specific_ids = []
+    if parseroutput is not None:
+        if len(parseroutput) == 0:
+            parseroutput = [parseroutput]
+        print("Extracting jobs with ids:{}".format(" ".join(parseroutput)))
+        for element in parseroutput:
+            if "-" not in element:
+                if RepresentsInt(element):
+                    specific_ids.append(int(element))
+                else:
+                    okquit(
+                        "Cannot parse {}, either specify an integer or a range".format(element))
+
+            else:
+                parts = element.split("-")
+                if len(parts) != 2 or not RepresentsInt(parts[0]) or not RepresentsInt(parts[1]):
+                    okquit(
+                        "Cannot parse {}, range should be of the form x-y".format(element))
+
+                startrange = int(parts[0])
+                endrange = int(parts[1])+1
+                if endrange < startrange:
+                    okquit("Range {} should start with smaller element".format(parts))
+                specific_ids.extend(list(range(startrange, endrange)))
+
+    return sorted(set(specific_ids))
+
+
+def parseSelector(parseroutput):
+
+    if parseroutput is not None:
+        parts = parseroutput.split(":")
+        pattern = "".join(parts[1:])
+        path = parts[0]
+        print("Extracting jobs with tag:\'{}\' and valuepattern:\'{}\''".format(
```

Comment:
You don't need to quote the `'` string when you are inside a `"` string, and the other way around

Reply:
In Python both `'` and `"` are used to construct strings